src/current/_includes/molt/replicator-flags.md

```diff
@@ -0,0 +1,85 @@
+### Replication flags
+
+The following flags are set with [`--replicator-flags`](#global-flags) and can be used in any [Fetch mode](#fetch-mode) that involves replication.
+
+|             Flag             |    Type    |                                                                  Description                                                                  |
+|------------------------------|------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| `--applyTimeout`             | `DURATION` | The maximum amount of time to wait for an update to be applied.<br><br>**Default:** `30s`                                                     |
+| `--dlqTableName`             | `IDENT`    | The name of a table in the target schema for storing dead-letter entries.<br><br>**Default:** `replicator_dlq`                                |
+| `--flushPeriod`              | `DURATION` | Flush queued mutations after this duration.<br><br>**Default:** `1s`                                                                          |
+| `--flushSize`                | `INT`      | Ideal batch size to determine when to flush mutations.<br><br>**Default:** `1000`                                                             |
+| `--gracePeriod`              | `DURATION` | Allow background processes to exit.<br><br>**Default:** `30s`                                                                                 |
+| `--logDestination`           | `STRING`   | Write logs to a file. If not specified, write logs to `stdout`.                                                                               |
+| `--logFormat`                | `STRING`   | Choose log output format: `"fluent"`, `"text"`.<br><br>**Default:** `"text"`                                                                  |
+| `--metricsAddr`              | `STRING`   | A `host:port` on which to serve metrics and diagnostics.                                                                                      |
+| `--parallelism`              | `INT`      | The number of concurrent database transactions to use.<br><br>**Default:** `16`                                                               |
+| `--quiescentPeriod`          | `DURATION` | How often to retry deferred mutations.<br><br>**Default:** `10s`                                                                              |
+| `--retireOffset`             | `DURATION` | How long to delay removal of applied mutations.<br><br>**Default:** `24h0m0s`                                                                 |
+| `--scanSize`                 | `INT`      | The number of rows to retrieve from staging.<br><br>**Default:** `10000`                                                                      |
```

Comment:
Reading down the alphabetical list of flags, I did not know what "staging" is referring to? After going down the list, I assume it is referring to `--stagingSchema`.  Is this a correct assumption?

Reply:
I clarified this description as it was 1) the least vague; 2) the first instance of "staging" in the table. I will do more work to contextualize "staging" in follow-up PRs that integrate Replicator into the MOLT Fetch docs.